README.md

```diff
@@ -43,16 +43,16 @@ as a part of the process.
 - Start the build:
 
   ```
-  yt build
+  make all
   ```
 
-The resulting microbit-micropython.hex file to flash onto the device can be
-found in the build/bbc-microbit-classic-gcc-nosd/source from the root of the
-repository.
+The resulting firmware.hex file to flash onto the device can be
+found in the build/ directory from the root of the repository.
 
-There is a Makefile provided that does some extra preprocessing of the source,
-which is needed only if you add new interned strings to qstrdefsport.h.  The
-Makefile also puts the resulting firmware at build/firmware.hex, and includes
+The Makefile provided does some extra preprocessing of the source,
+it adds version information to the UICR region, and process new interned
+strings to qstrdefsport.h.
```

Comment:
The last part of the sentence here about qstrdefsport.h is no longer true, so can be removed.

Reply:
👍 
Commit updated.